webfront/searcher/elastic_controller.py

```diff
@@ -62,12 +72,25 @@ def __init__(self, queryset_manager=None):
         self.index = settings.SEARCHER_INDEX
         self.queryset_manager = queryset_manager
         self.headers = {"Content-Type": "application/json"}
+        using_auth = settings.SEARCHER_USER != ""
+        if using_auth:
+            self.headers["Authorization"] = basic_auth(
+                settings.SEARCHER_USER, settings.SEARCHER_PASSWORD
+            )
         if proxy is not None and proxy != "":
             proxy = urllib.parse.urlparse(proxy)
-            self.connection = http.client.HTTPConnection(proxy.hostname, proxy.port)
+            self.connection = HTTPSConnection(
+                proxy.hostname,
+                proxy.port,
+                context=ssl._create_unverified_context() if using_auth else None,
```

Comment:
I think that the unverified context is required because we use `https`, not because we use authentication:

```suggestion
                context=ssl._create_unverified_context() if url.scheme == "https" else None,
```